apps/infrastructure/src/modules/chatbot/lambda_chatbot.tf

```diff
@@ -18,7 +18,7 @@ module "lambda_function" {
   package_type  = "Image"
   architectures = ["x86_64"]
 
-  image_uri = module.ecr.repository_url
+  image_uri = "${module.ecr.repository_url}:latest"
```

Comment:
Maybe we couldn't use "latest" tag, because we can't use any cached version of the image.

Reply:
As discussed on slack, we'll stick to the usage of `latest` tag until a more thoughtful way is implemented